src/main/java/org/terasology/biomesAPI/BiomeRegistry.java

```diff
@@ -82,6 +83,16 @@ default void setBiome(Biome biome, Chunk chunk, Vector3ic pos) {
      */
     Optional<Biome> getBiome(int x, int y, int z);
 
+    /**
+     * Gets biome at position in chunk view.
+     *
+     * @param relX x position of the block to get biome of, relative to the chunk view
+     * @param relY y position of the block to get biome of, relative to the chunk view
+     * @param relZ z position of the block to get biome of, relative to the chunk view
+     * @return Biome of the block
+     */
+    Optional<Biome> getBiome(ChunkView view, int relX, int relY, int relZ);
```

Comment:
Should there be a `set` method of this form for symmetry?

Reply:
Maybe - there are already `setBiome` methods operating on a `Chunk`, though, without corresponding `getBiome`s.